### PR TITLE
Use chrome icons

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Changes
- no visual changes
- uses chrome static assets for landing page icons

## This should be merged only after current beta version of chrome is in prod.

To test this locally tou will need to run chrome locally as well to provide correct baseUri (`/` or `/beta/`)

### CODE REVIEW

**Comment by Hyperkid123** [comment]
cc @karelhala

**Comment by karelhala** [comment]
Let's postpone merging of this issue once we have these icons all the way in prod-stable of chrome so we don't break anything.

File: src/consts.js [hunk 1]
```diff
@@ -53,7 +49,7 @@ export const activeTechnologies = [
             compliance: '/compliance',
             'drift analysis': '/drift'
         },
-        image: hcm,
+        image: `${document.baseURI}apps/chrome/assets/images/platform-icons/rhel-cs-namespace.svg`,
```
[the remark this answers is not in the document]
Comment (ryelo) [reply src/consts.js]: The `Cloud Management Services for Red Hat Enterprise Linux` should be: https://github.com/RedHatInsights/insights-chrome/blob/master/static/images/platform-icons/hcm-namespace.svg

the `Hybrid Cloud Management Services` should be: https://github.com/RedHatInsights/insights-chrome/blob/master/static/images/platform-icons/rhel-cs-namespace.svg

**Comment by Hyperkid123** [comment]
@ryelo thanks I will swap the icons names so it actually matches the cards.

**Comment by Hyperkid123** [comment]
@ryelo the icons should be swapped once https://github.com/RedHatInsights/insights-chrome/pull/405 is merged.

**Comment by ryelo** [comment]
@karelhala Should we tag this on hold until the next chrome stable push?

**Comment by karelhala** [comment]
@Hyperkid123 :wink:

**Pull request merged**